lib/form_data.js

```diff
@@ -214,10 +214,28 @@ FormData.prototype._multiPartHeader = function(field, value, options) {
   return '--' + this.getBoundary() + FormData.LINE_BREAK + contents + FormData.LINE_BREAK;
 };
 
+FormData.prototype._encodeHeaderParam = function(str, lang) {
+  if (!lang || typeof lang !== 'string') {
+    lang = '';
+  }
+
+  // taken from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
+  // @note see RFC 8187
+  return 'UTF-8\'' + lang + '\'' + encodeURIComponent(str).
+    // Note that although RFC3986 reserves "!", RFC8187 does not,
+    // so we do not need to escape it
+    replace(/['()]/g, escape). // i.e., %27 %28 %29
+    replace(/\*/g, '%2A').
+    // The following are not required for percent-encoding per RFC8187,
+    // so we can allow for a little better readability over the wire: |`^
+    replace(/%(?:7C|60|5E)/g, unescape);
+};
+
 FormData.prototype._getContentDisposition = function(value, options) {
 
   var filename
-    , contentDisposition
+    , contentDisposition = []
+    , NON_ASCII_REGEX = /[^\x20-\x7e\xa0-\xff]/g
```

Comment:
Use regex used by node to validate header: /[^\t\x20-\x7e\x80-\xff]/
Refer: https://github.com/nodejs/node/blob/v13.6.0/lib/_http_common.js#L214